synclet/deploy_utils.py

```diff
@@ -0,0 +1,36 @@
+# CONSTANTS
+DEFAULT_TEMPLATE = 'synclet/synclet-conf.template.yaml'
+DEFAULT_DOCKERFILE = 'Dockerfile.synclet'
+
+ENV_DEVEL = 'devel'
+ENV_PROD = 'prod'
+
+ENV_TO_PROJ = {
+    ENV_DEVEL: 'blorg-dev',
```

Comment:
are these worth changing?

Reply:
ehhh, that's the cluster name. i suppose we could make a new cluster for all this testing stuff, but 🤷‍♀️